meson.build

```diff
@@ -1,24 +1,43 @@
+# SPDX-License-Identifier: <<spdxid>>
+# Copyright © 2024 Dylan Baker
```

Comment:
@dcbaker Was this addition intentional?

Reply:
No, sigh. My editor is trying to be helpful at least.